executor/executor_test.go

```diff
@@ -1022,6 +1022,50 @@ func (s *testSuite) TestJSON(c *C) {
 	result.Check(testkit.Rows(`3 {} <nil>`))
 }
 
+func (s *testSuite) TestGeneratedColumnDDL(c *C) {
```

Comment:
moved into ddl_db_test.go. BTW, I think the test cases is enough here, because all feature added by this PR are just two:
1. desc or show create table with generated column;
2. if the generated column refers illegal columns, throw error.